### PR TITLE
rtt updates

### PR DESCRIPTION
**Pull request by terrapin47**
Changes made using the realtime translation.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -892,29 +892,29 @@ QUEST_LV_0100_20150317_000891	You need the seal of space to enter the hidden san
...
 QUEST_LV_0100_20150317_000893	The most powerful barrier is in the 2nd floor. {nl} I heard that it can exert the greatest force when the divine sphere is there.
 QUEST_LV_0100_20150317_000894	Gesti made the first move. {nl} It will be difficult to approach secretly.
-QUEST_LV_0100_20150317_000895	I must change the plan. {nl} First you need to go watch the movement of the Gesti. {nl}
+QUEST_LV_0100_20150317_000895	I must change the plan. {nl} First you need to go watch Gesti's actions. {nl}
 QUEST_LV_0100_20150317_000896	Let's start by occupying the bell tower.
 QUEST_LV_0100_20150317_000897	From here, we can see what Gesti is going to do.
 QUEST_LV_0100_20150317_000898	Alright, bring the seal of space from Sventove Central Altar. {nl} You can do it, right?
-QUEST_LV_0100_20150317_000899	You must know that the revelation is hidden in the sanctuary. {nl} The seal of the space is the only key that get you to the sanctuary. {nl}
+QUEST_LV_0100_20150317_000899	You must know that the revelation is hidden in the sanctuary. {nl} The seal of the space is the only key that will get you to the sanctuary. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: The seal of the space -> The crest of space

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -892,29 +892,29 @@ QUEST_LV_0100_20150317_000891	You need the seal of space to enter the hidden san
...
 QUEST_LV_0100_20150317_000905	So you activated the Malda altar? {nl} Now it will be safe to stay here for the time being. Good work.
-QUEST_LV_0100_20150317_000906	Demons are going around the second floor searching every corner. {nl} Activating the Auka altar will be able to turn their attention.
+QUEST_LV_0100_20150317_000906	Demons are going around the second floor searching every corner. {nl} Activating the Auka altar will catch their attention.
 QUEST_LV_0100_20150317_000907	{memo X}I'm relieved you're beside me. {nl} I think it was a good thing that Laima chose you.
-QUEST_LV_0100_20150317_000908	Good job. {nl} When this is over, I should gather all brothers and drive them away at once.
-QUEST_LV_0100_20150317_000909	I'll ring the bell and let you know when Gesti show up so have a safe trip on your way.
+QUEST_LV_0100_20150317_000908	Good job. {nl} When this is over, I should gather all the brothers and drive them away at once.
+QUEST_LV_0100_20150317_000909	I'll ring the bell and let you know when Gesti shows up so have a safe trip on your way.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Add a comma before 'so'?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -892,29 +892,29 @@ QUEST_LV_0100_20150317_000891	You need the seal of space to enter the hidden san
...
 QUEST_LV_0100_20150317_000911	May the blessings of the Goddess be with you.
 QUEST_LV_0100_20150317_000912	So Gesti just ran away. {nl} Fine. Since you got the revelation, we've completed the mission. {nl}
 QUEST_LV_0100_20150317_000913	{memo X}
-QUEST_LV_0100_20150317_000914	When you find the revelation, tell Mater Paladin of the story you've been through so far. {nl} May the blessings of the Goddess be with you.
+QUEST_LV_0100_20150317_000914	When you find the revelation, tell the Paladin Master of the story you've been through so far. {nl} May the blessings of the Goddess be with you.
 QUEST_LV_0100_20150317_000915	Seems like Gesti did not notice yet. {nl}Better prepare the divine crystal.
-QUEST_LV_0100_20150317_000916	All the preparations made. {nl} Barriers of the church will soon weaken the Gesti. {nl}
-QUEST_LV_0100_20150317_000917	The weakened Gesti will be enough for you to handle. {nl} When the Gesti is cornered, I will make the final strike with the divine sphere.
+QUEST_LV_0100_20150317_000916	All the preparations have been made. {nl} The barriers of the church will soon weaken Gesti. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: church -> chapel

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -892,29 +892,29 @@ QUEST_LV_0100_20150317_000891	You need the seal of space to enter the hidden san
...
+QUEST_LV_0100_20150317_000917	Once weakened, Gesti will be enough for you to handle. {nl} When Gesti is cornered, I will make the final strike with the divine sphere.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: divine sphere -> Divine Sphere?

**Comment by terrapin47** [comment]
Updated based on feedback.

File: ITEM.tsv [hunk 5]
```diff
@@ -1978,7 +1978,7 @@ ITEM_20150317_001977	Grinender Card
 ITEM_20150317_001978	Glass Mole Card 
 ITEM_20150317_001979	Ginklas Card 
 ITEM_20150317_001980	Netherbovine Card 
-ITEM_20150317_001981	Necrovanter Card 
+ITEM_20150317_001981	Necroventer Card
```
Comment (Sunaries) [inline ITEM.tsv]: Necroventer -> Necroventor

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -890,54 +890,54 @@ QUEST_LV_0100_20150317_000889	Strange, it was definitely working.
...
 QUEST_LV_0100_20150317_000900	It's not just the Tenet Cathedral. {nl} You can enter all the places that the Goddess hid.
 QUEST_LV_0100_20150317_000901	The seal of space can only be used by the Revelator so you won't be able to find the revelation right away. {nl}
 QUEST_LV_0100_20150317_000902	But the enemy is the Demon King. Even the Paladin Master couldn't defeat him.{nl}I should use the power of Broken Altar.
 QUEST_LV_0100_20150317_000903	We can't do anything about the destroyed altar but the fragments still have power. {nl} Gather the pieces and insert it in the 8 pillars of the Sventove Central Hall.
-QUEST_LV_0100_20150317_000904	When you are done, I will be able to tie up Gesti through the power of the divine crystal and the church. {nl} It may be for a short while but that's the best option we have.
+QUEST_LV_0100_20150317_000904	When you are done, I will be able to restrict Gesti through the power of the divine crystal and the chapel. {nl} It may be for a short while but that's the best option we have.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: divine crystal -> Divine Sphere

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -2332,16 +2332,16 @@ QUEST_LV_0100_20150317_002331	Cyrenia Odell
...
 QUEST_LV_0100_20150317_002333	Ah? Woman. I forgot.{nl}I will release you as promised so make her alive.
 QUEST_LV_0100_20150317_002334	Sincerity is our strength. {nl} There will be no problems if you sincerely believe and follow the Goddess.
-QUEST_LV_0100_20150317_002335	I think being confined in there caused my throat and now my throat hurts. {nl} But what about the girl who was together? Where did she go?
+QUEST_LV_0100_20150317_002335	I think being confined in there damaged my throat and now my throat hurts. {nl} But what about the girl who was with us? Where did she go?
 QUEST_LV_0100_20150317_002336	Aren't you the Revelator who saved us last time? {nl} We were more than glad to see you back then.
 QUEST_LV_0100_20150317_002337	The Goddess must have sent you.{nl}If that's not it.. we already.. It's too scary to think about it.
-QUEST_LV_0100_20150317_002338	It was horrible in captivity. {nl} Vubbes come and smell us.. {nl} and make us dig here and there using pickax..
+QUEST_LV_0100_20150317_002338	It was horrible in captivity. {nl} Vubbes would come and smell us.. {nl} and make us dig here and there using a pickaxe..
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Vubbes -> The Vubbes

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -5360,7 +5360,7 @@ QUEST_LV_0100_20150317_005359	A little vandalism on the summon circle absorbed b
 QUEST_LV_0100_20150317_005360	Go to Alfonsas
 QUEST_LV_0100_20150317_005361	{memo X}
 QUEST_LV_0100_20150317_005362	Chase Gesti
-QUEST_LV_0100_20150317_005363	Arrived at the church courtyard and saw Gesti entering the church. Defeat the Chapparition and follow Gesti.
+QUEST_LV_0100_20150317_005363	Arrived at the chapel courtyard and saw Gesti entering the chapel. Defeat the Chapparition and follow Gesti.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: at the chapel courtyard -> at Temple Courtyard

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -6629,7 +6629,7 @@ QUEST_LV_0100_20150414_006628	I thinks monsters are crawling out of the mysterio
 QUEST_LV_0100_20150414_006629	Maintenance would be easier now that the barrier is gone.
 QUEST_LV_0100_20150414_006630	Go down from the middle of Twin Bridge and you'll arrive at Srautas Gorge. {nl}Go further into the left from there and you'll get to Gele Plateau.
 QUEST_LV_0100_20150414_006631	First, defeat the Hanamings around here. {nl}And gather Hanaming Petals while you're at it.
-QUEST_LV_0100_20150414_006632	Miners' Village is a problem but we also haven't got a rest from the endless battle. {nl}Solving those Chupacabra would make it a lot better.
+QUEST_LV_0100_20150414_006632	The Miners' Village is a problem but we also haven't gotten any rest from the endless battle. {nl}Defeating those Chupacabra would make it a lot better.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: those Chupacabra -> those Chupacabras?

**Comment by terrapin47** [comment]
Updated :sake:

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**